lib/tmuxinator/config.rb

```diff
@@ -4,18 +4,34 @@ class Config
     NO_LOCAL_FILE_MSG = "Project file at ./.tmuxinator.yml doesn't exist."
 
     class << self
-      def root
-        root_dir = File.expand_path("#{ENV['HOME']}/.tmuxinator")
-        Dir.mkdir(root_dir) unless File.directory?(root_dir)
-        root_dir
+      # The directory (created if needed) in which to store new projects
+      def directory
+        environment = ENV["TMUXINATOR_CONFIG"]
```

Comment:
Thanks for the thorough review, I'll get on to these in a couple of days to allow for more reviewers to have their say in the meantime.